modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/LiferayBasePlugin.java

```diff
@@ -14,7 +14,6 @@
 
 package com.liferay.gradle.plugins;
 
-import com.liferay.gradle.plugins.NodeDefaultsPlugin;
```

Comment:
Instead of creating `Auto SF` commits like this, may I suggest to squash them into the commits that originally added this code? This way, the history will be cleaner 😊